.github/workflows/stencil-nightly.yml

```diff
@@ -24,7 +29,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/workflows/actions/build-core-stencil-prerelease
         with:
-          stencil-version: nightly
+          stencil-version: ${{ inputs.npm_release_tag || 'nightly' }}
```

Comment:
We need these conditionals to default to 'nightly' for the cron job, since there's no 'input' to those

Reply:
Wouldn't `npm_release_tag` default to "nightly" because of how you configured the input?

Reply:
I _think_ it wouldn't, since that input is tied to the `workflow_dispatch` event (and aren't shared to the cron).

Reply:
That said, we can back this out and see what happens tonight 😆

Reply:
Ahh no you're right -- cron dispatches don't have access to inputs according to https://dev.to/mrmike/github-action-handling-input-default-value-5f2g